src/Microsoft.Health.CosmosDb/Features/Health/HealthCheckResponseWriter.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net.Mime;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.CosmosDb.Features.Health
+{
+    public static class HealthCheckResponseWriter
```

Comment:
I think we can safely move this class to the API project.
Then remove the reference from API[Cosmos]. The api project shouldn't have a reference to a specific database implementation

Reply:
Agree. I think I will just make it an anonymous method instead of having a separate class/method for the response writer. We can refactor it later if we the case for re-using it comes up.